packages/server/lib/browsers/firefox-util.ts

```diff
@@ -41,7 +43,7 @@ const getDelayMsForRetry = (i) => {
     return 500
   }
 
-  if (i < 63) {
+  if (i < maxRetries) {
```

Comment:
```suggestion
  if (i <= maxRetries) {
```
looks like we lost a retry here

Reply:
Good Catch!. I commited your suggestion